### PR TITLE
v2.9.1

### PR DESCRIPTION
**Pull request by dliu-workos**
Bumping to v2.9.1 for release:

- https://github.com/workos/workos-node/pull/643

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,4724 +1,8 @@
 {
   "name": "@workos-inc/node",
-  "version": "2.8.1",
-  "lockfileVersion": 2,
+  "version": "2.9.1",
+  "lockfileVersion": 1,
```
Comment (robframpton) [inline package-lock.json:4]: Any idea why this went from version `2` to `1`?

Comment (dliu-workos) [reply package-lock.json:4]: I think someone might've ran `npm i` using NodeV16.

Comment (dliu-workos) [reply package-lock.json:4]: Someone is renovate-bot: https://github.com/workos/workos-node/pull/641/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R4

Comment (robframpton) [reply package-lock.json:4]: We should lock it down so that renovate uses the same version as our other projects.

**Review by robframpton — APPROVED** [review]

**Pull request merged**